code-challenges.js

```diff
@@ -107,34 +110,86 @@
+// Now that we have an array to work with, we can use .reverse() method to reverse the elements in our array.
+// The elements in the array are now reversed, however they are all seperated by single quotes since they are elements of an array.
+// Lastly, we need to Console log the variable to check the result.
 
 //Find a method that will convert the array back into a string
 
-    // The .join("") method will do just that.
+// const bestCohortof2022ever = "Golf 2022"
+// console.log(bestCohortof2022ever.split(""))
 
 // Console log the variable to check the result.
 
 // const currentCohort = "Golf 2022"
 // // Expected output: "2202 floG"
 
 
-
 // const bestCohortof2022ever = currentCohort.split('').reverse().join("")
 // const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
 
 
+// --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
+// Describe your process:
+    
+    // 1. Find a method that will go through the array and grab the last of that element we pass in. 
+    
+    // 2. We will Use the .lastIndexOf() method on the array and pass the arguments to Return the last indexs value. This is useful 
+    // when there are repeated values in an array.
+    
+    // 3. We will do all this witin the console.log when calling on out array to pull the desired index.
+    
+        // The ouput should show us the last index or location of the number we pass in the array.
+    
+    
+        // const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+    
+        // console.log(myNumbers.lastIndexOf(10))  
+        // output: 8
+        
+        // console.log(myNumbers.lastIndexOf(42))
+        // //output: 7
+//Rant: 
+// Aftergoing through this assesment I've realized I keep trying to remember everything and thats not going to happen. I'm familiar with the process and a good portion of the vocabulary. However, I still cannot recall the proper syntax to ultalize I have to look it up.
+
+
+
+
+// --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+    
+    // Describe your process:
+    
+    //I need Find an array method that sorts. In this case, the method I will use is .sort()
+    // /When we utalize /Since .sort() sorts numbers from smallest to largest, so we must find a method that reverses the elementss in an array to make the numbers sort   from largest to smallest. In this case the method we can use is .reverse()
+    //We can do all this by staying within the console.log.
+    // Example:  console.log(myArrayList.sort().reverse())
+    //This code tells the interpreter to first sort the elements in the array and then reverse the order of that sorted list.
+    // Question: Can we can chain these two methods togeather. I have no idea how that looks. However, I 
```

Comment:
Looks like you got cut off, but successfully chained two methods together below.  Good work!